packages/calcite-components/src/components/panel/panel.tsx

```diff
@@ -525,27 +511,16 @@ export class Panel
   };
 
   renderContent(): VNode {
-    const { hasFab } = this;
-
-    const defaultSlotNode: VNode = <slot key="default-slot" />;
-    const containerNode = hasFab ? (
-      <section class={CSS.contentContainer}>{defaultSlotNode}</section>
```

Comment:
@asangma do you think you could review these changes? I changed this because it was causing some problems with `onSlotChange` since this default slot was getting moved into a section element depending on if there was a FAB or not. I don't think its necessary if we always have the section element present and the FAB content classes are no longer conditional.